### PR TITLE
refactor(web): Remove logic from operations controller into service class

### PR DESCRIPTION
**Pull request by robzienert**
Just some cleanup before I do some other work. Also some annotations floating around on some interfaces as I figure out what I need to do and I'm figuring out the implicit contracts of these interfaces.

### CODE REVIEW

File: clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperationConverterNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.clouddriver.orchestration;
```
Comment (scottfrederick) [inline clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperationConverterNotFoundException.java]: Missing copyright

File: clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperationException.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package com.netflix.spinnaker.clouddriver.orchestration;
```
Comment (scottfrederick) [inline clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperationException.java]: Missing copyright

File: clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperationNotFoundException.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.netflix.spinnaker.clouddriver.orchestration;
```
Comment (scottfrederick) [inline clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperationNotFoundException.java]: Missing copyright

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy [hunk 4]
```diff
@@ -49,165 +31,53 @@ import org.springframework.web.bind.annotation.RestController
...
 
-  @Autowired DescriptionAuthorizer descriptionAuthorizer
-
-  /*
-   * APIs
-   * ----------------------------------------------------------------------------------------------------------------------------
-   */
-
-  /**
+  @Autowired
```
Comment (scottfrederick) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy]: `@Autowired` isn't required on ctors on Java classes, is it required for Groovy?

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy [hunk 5]
```diff
@@ -49,165 +31,53 @@ import org.springframework.web.bind.annotation.RestController
...
   @Deprecated
   @RequestMapping(value = "/ops", method = RequestMethod.POST)
-  Map<String, String> operations(@RequestParam(value = "clientRequestId", required = false) String clientRequestId,
-                                 @RequestBody List<Map<String, Map>> requestBody) {
-    List<AtomicOperation> atomicOperations = collectAtomicOperations(requestBody)
-    start(atomicOperations, clientRequestId)
+  OperationsService.StartOperationResult operations(
+    @RequestParam(value = "clientRequestId", required = false) String clientRequestId,
+    @RequestBody List<Map<String, Map>> requestBody) {
+    List<AtomicOperation> atomicOperations = operationsService.collectAtomicOperations(requestBody)
```
Comment (scottfrederick) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy]: Calling one method of `operationsService` to get a value (`atomicOperations`), just to pass that value to another method of the same service, is a bad smell. The pattern is repeated several times in this class. 

Can both calls be encapsulated into one method? Or is that an indication that `collectAtomicOperations` and `start` should belong to separate services?

Comment (robzienert) [reply clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy]: Good point. Thank you - I'll split these into two services. I wasn't too pleased with this myself, thanks for keeping me honest. :)

**Pull request closed without merging**